.ci/generate_bear_requirements.py

```diff
@@ -163,6 +164,13 @@ def get_cabal_requirements(requirements):
     return _get_requirements(requirements, '==')
 
 
+def get_languages(bears):
+    language_dict={}
+    for bear in bears:
+        language_dict[str(bear.name)]=bear.LANGUAGES
```

Comment:
redundant `str()` ?

Reply:
I think it returns a string literal. Correct me if I am wrong.

Reply:
When i coded the solution and tested  it gave me an error so i put it there.